### ISSUE TITLE
cannot run example from new clone

### ISSUE THREAD

**Issue by kelvin-dev** [react-native-mapbox-gl/maps#1689]
Dear team, please help investigate the logs,

```error: Error: Unable to resolve module AccessibilityInfo from D:\Demos\maps-2\node_modules\react-native\Libraries\react-native\react-native-implementation.js: AccessibilityInfo could not be found within the project or in these directories:
  ..\node_modules\react-native\node_modules
  ..\node_modules
  D:\Demos\maps-2\example\node_modules\AccessibilityInfo

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
  17 | module.exports = {
  18 |   // Components
> 19 |   get AccessibilityInfo() {
     |       ^
  20 |     return require('AccessibilityInfo');
  21 |   },
  22 |   get ActivityIndicator() {
    at ModuleResolver.resolveDependency (D:\Demos\maps-2\example\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:129:15)
    at DependencyGraph.resolveDependency (D:\Demos\maps-2\example\node_modules\metro\src\node-haste\DependencyGraph.js:288:43)
    at Object.resolve (D:\Demos\maps-2\example\node_modules\metro\src\lib\transformHelpers.js:129:24)
    at resolve (D:\Demos\maps-2\example\node_modules\metro\src\DeltaBundler\traverseDependencies.js:396:33)
    at D:\Demos\maps-2\example\node_modules\metro\src\DeltaBundler\traverseDependencies.js:412:26
    at Array.reduce (<anonymous>)
    at resolveDependencies (D:\Demos\maps-2\example\node_modules\metro\src\DeltaBundler\traverseDependencies.js:411:33)
    at processModule (D:\Demos\maps-2\example\node_modules\metro\src\DeltaBundler\traverseDependencies.js:140:31)
    at async addDependency (D:\Demos\maps-2\example\node_modules\metro\src\DeltaBundler\traverseDependencies.js:230:18)
    at async Promise.all (index 3)
```

```
System:
  Binaries:
    Node: 14.17.4 - D:\Program Files\nodejs10\node.EXE
    Yarn: 1.22.4 - D:\Roaming\npm\yarn.CMD
    npm: 6.14.5 - D:\Roaming\npm\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK:
      Android NDK: 18.1.5063045
  IDEs:
    Android Studio: Not Found
  Languages:
    Java: 14.0.1 - /cygdrive/c/Program Files/OpenJDK/jdk-14.0.1/bin/javac
    Python: 3.8.2 - /cygdrive/d/Program Files (x86)/Python38-32/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.66.0 => 0.66.0
  npmGlobalPackages:
    *react-native*: Not Found
```
